userspace/libsinsp/container_docker.h

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright (C) 2013-2018 Draios Inc dba Sysdig.
```

Comment:
Nit: My understanding is we're now (within the last week or two) officially "Sysdig".

Reply:
Oh neat, I didn't know that! If we do change it to Sysdig, we should tackle that separately and change all the headers in one step.